packages/api-browser/src/screen-snippet.js

```diff
@@ -1,4 +1,5 @@
 /* globals html2canvas */
+import 'html2canvas';
```

Comment:
I've tried to, but I don't think so... html2canvas's package.json has a `main` property which references a generated UMD bundle, but it doesn't have a `module` or `jsnext` properties.

There might be a way to tell Rollup to pick up `src/core.js` in html2canvas  (where it is exported), then build it from there, but I can't see how to do that (not even sure if it's possible) - I've had no luck so far... if you can see a way to do it, then please let me know!